File: Compendium.Admin.WebApi/Models/Item.cs
using System;

namespace Compendium.Admin.WebApi.Models
{
    public class Item
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public string Subject { get; set; }
        public string Type { get; set; }
        public bool IsPublic { get; set; }
        public bool IsValid { get; set; }
        public string Creator { get; set; }
        public string Contributers { get; set; } //New table?
        public string Coverage { get; set; }
        public DateTime Date { get; set; }
        public string Format { get; set; }
        public string Identifier { get; set; }
        public string Language { get; set; }
        public string Publisher { get; set; }
        public string Relation { get; set; }
        public string Rights { get; set; }  //New table?
        public string Source { get; set; }
        public string Abstract { get; set; }
        public string AccessRights { get; set; }
        public string AccrualMethod { get; set; }
        public string AccrualPeriodicity { get; set; }
        public string Alternative { get; set; }
        public string Audience { get; set; }
        public bool Available { get; set; }
        public string BibliographicCitation { get; set; }
        public string ConformsTo { get; set; }
        public string Created { get; set; }
        public DateTime DateAccepted { get; set; }
        public DateTime DateCopyrighted { get; set; }
        public DateTime DateSubmitted { get; set; }
        public DateTime DateCreatedOn { get; set; }
        public DateTime DateModifiedOn { get; set; }
        public string EducationLevel { get; set; }
        public string InstructionalMethod { get; set; }
        public string Extent { get; set; }
        public bool HasFormat { get; set; }
        public bool HasPart { get; set; }
        public bool HasVersion { get; set; }
        public string IsFormatOf { get; set; }
        public string IsPartOf { get; set; }
        public string IsReferencedBy { get; set; }
        public string IsReplacedBy { get; set; }
        public string IsRequiredBy { get; set; }
        public string Issued { get; set; }
        public string IsVersionOf { get; set; }
        public string License { get; set; }
        public string Mediator { get; set; }
        public string Medium { get; set; }
        public string Modified { get; set; }        //Is there a difference between this and DateModifiedOn?
        public string Provenance { get; set; }
        public string References { get; set; }      //New Table?
        public string Replaces { get; set; }        //New Table?
        public string Requires { get; set; }        //New Table
        public string RightsHolder { get; set; }
        public string Spatial { get; set; }
        public string TableOfContents { get; set; }
        public string Temporal { get; set; }
    }
}

File: server/models/Item.js
const Sequelize = require('sequelize');
const db = require('../db/dbConnection');

const Item = db.define('item', {
    title: {
      type: Sequelize.STRING
    },
    subject: {
      type: Sequelize.STRING
    },
    type: {
        type: Sequelize.JSON
      },
    description: {
        type: Sequelize.STRING
    },
    valid: {
        type: Sequelize.BOOLEAN
    },
    creator: {
        type: Sequelize.STRING
    },
    contributers: {
        type: Sequelize.JSON
    },
    coverage: {
        type: Sequelize.STRING
    },
    date: {
        type: Sequelize.DATE
    },
    format: {
        type: Sequelize.STRING
    },
    identifier: {
        type: Sequelize.STRING
    },
    language: {
        type: Sequelize.STRING
    },
    publisher: {
        type: Sequelize.STRING
    },
    relation: {
        type: Sequelize.STRING
    },
    rights: {
        type: Sequelize.STRING
    },
    source: {
        type: Sequelize.STRING
    },
    abstract: {
        type: Sequelize.STRING(2500)
    },
    accessRights: {
        type: Sequelize.STRING
    },
    accrualMethod: {
        type: Sequelize.STRING
    },
    accrualPeriodicity: {
        type: Sequelize.STRING
    },
    alternative: {
        type: Sequelize.STRING
    },
    audience: {
        type: Sequelize.STRING
    },
    available: {
        type: Sequelize.BOOLEAN
    },
    bibliographicCitation: {
        type: Sequelize.STRING
    },
    conformsTo: {
        type: Sequelize.STRING
    },
    created: {
        type: Sequelize.STRING
    },
    dateAccepted: {
        type: Sequelize.DATE
    },
    dateCopyrighted: {
        type: Sequelize.DATE
    },
    dateSubmitted: {
        type: Sequelize.DATE
    },
    educationLevel: {
        type: Sequelize.STRING
    },
    extent: {
        type: Sequelize.STRING
    },
    hasFormat: {
        type: Sequelize.STRING
    },
    hasPart: {
        type: Sequelize.STRING
    },
    hasVersion: {
        type: Sequelize.STRING
    },
    instructionalMethod: {
        type: Sequelize.STRING
    },
    isFormatOf: {
        type: Sequelize.STRING
    },
    isPartOf: {
        type: Sequelize.STRING
    },
    isReferencedBy: {
        type: Sequelize.STRING
    },
    isReplacedBy: {
        type: Sequelize.STRING
    },
    isRequiredBy: {
        type: Sequelize.STRING
    },
    issued: {
        type: Sequelize.STRING
    },
    isVersionOf: {
        type: Sequelize.STRING
    },
    license: {
        type: Sequelize.STRING
    },
    mediator: {
        type: Sequelize.STRING
    },
    medium: {
        type: Sequelize.STRING
    },
    modified: {
        type: Sequelize.STRING
    },
    provenance: {
        type: Sequelize.STRING
    },
    references: {
        type: Sequelize.STRING
    },
    replaces: {
        type: Sequelize.STRING
    },
    requires: {
        type: Sequelize.STRING
    },
    rightsHolder: {
        type: Sequelize.STRING
    },
    spatial: {
        type: Sequelize.STRING
    },
    tableOfContents: {
        type: Sequelize.STRING
    },
    temporal: {
        type: Sequelize.STRING
    },
    collectionID: {
        type: Sequelize.INTEGER
    }
});

module.exports = Item;